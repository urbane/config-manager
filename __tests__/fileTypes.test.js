import {getUpdatedConfig} from '../index'


test('Test getUpdatedConfig for YAML.', () => {
    let newConfig = getUpdatedConfig(__dirname + '/files/yaml.yml', {
        newRootProp: 'newVal'
    })
    expect(newConfig).toMatchSnapshot();
});