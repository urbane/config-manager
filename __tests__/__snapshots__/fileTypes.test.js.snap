// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test getUpdatedConfig for YAML. 1`] = `Promise {}`;
